oar/core/advisory.py

```diff
@@ -252,7 +252,7 @@ def check_cve_tracker_bug(self):
             AdvisoryException: error when invoke elliott cmd
 
         Returns:
-            json: missed CVE tracker bugs
+            List[str]: missed CVE tracker bugs
```

Comment:
The method returns a different type.

Reply:
To clarify it a little bit. It's not necessary wrong and it is only a documentation, so we potentially could use also the one you used. However, it's better to align the changes with the rest of the file and use standard built-in type.